### ISSUE TITLE
 Could you please help to indicate which table that the script will produce?

### ISSUE THREAD

**Issue by xuuuluuu** [hsukyle/cactus-maml#11]

**Comment by xuuuluuu** [comment]
Hi Kyle,

Thanks for your intense work. Could you please help to indicate which table that the script will produce, like what you did in "miniimagenet_kmeans.sh" script. 

#!/usr/bin/env bash
# this run reproduces the second-to-last line of Table 4 of the paper.

And I tried to run "omniglot_kmeans.sh" script, the program stucks...

WARN: Cannot define MaxPoolGrad, likely already defined for this version of tensorflow: "Registering two gradient with name 'MaxPoolGrad'! (Previous registration was in _find_and_load_unlocked <frozen importlib._bootstrap>:957)"
0.0
2019-04-10 13:17:18.849101: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Encoder: acai
Setting up tasks for meta-training
Generating indices for 240000 tasks
Using 1 k-means based partition(s) of encoding space to create classes
Number of encodings: 1, number of n_clusters: 1, number of inits:  1
get_partitions_kmeans_n_clusters:   0%|                                                           | 0/1 [00:00<?, ?it/s]
get_partitions_kmeans_encodings:   0%|                                                            | 0/1 [00:00<?, ?it/s]

**Comment by kylehkhsu** [comment]
I've added comments to the scripts like you suggested. The output suggests the program is running the k-means pre-processing step. Did you give it a few minutes to do so?

**Comment by kylehkhsu** [comment]
Closing, feel free to re-open should you have additional concerns.
